collections/m2m gateway/tenant/Get Tenants.bru

```diff
@@ -0,0 +1,21 @@
+meta {
+  name: Tenants
```

Comment:
Minor: for consistency with https://github.com/pagopa/interop-be-monorepo/pull/1831 (already merged) I would name this `List tenants` (same for the filename)